.github/workflows/push.yml

```
@@ -212,3 +211,5 @@ jobs:
           body: |
             This is an automated release, triggered by a recent push. 
             This may or may not be stable, so please have a look at the stable release(s).
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (2 > 0)

(empty-lines)